easybuild/easyblocks/p/python.py

```diff
@@ -279,6 +279,21 @@ def install_step(self):
         if self.cfg['ebpythonprefixes']:
             write_file(os.path.join(self.installdir, self.pythonpath, 'sitecustomize.py'), SITECUSTOMIZE)
 
+        # symlink lib/python*/lib-dynload to lib64/python*/lib-dynload if it doesn't exist;
+        # see https://github.com/easybuilders/easybuild-easyblocks/issues/1957
+        lib_dynload = 'lib-dynload'
+        lib_dynload_path = os.path.join(self.installdir, 'lib', 'python%s' % self.pyshortver, lib_dynload)
+        if not os.path.exists(lib_dynload_path):
+            lib64_dynload_path = os.path.join(self.installdir, 'lib64', 'python%s' % self.pyshortver, lib_dynload)
+            if os.path.exists(lib64_dynload_path):
+                lib_dynload_parent = os.path.dirname(lib_dynload_path)
+                mkdir(lib_dynload_parent, parents=True)
+                cwd = change_dir(lib_dynload_parent)
+                # use relative path as target, to avoid hardcoding path to install directory
+                target_lib_dynload = os.path.join('..', '..', 'lib64', 'python%s' % self.pyshortver, lib_dynload)
+                symlink(target_lib_dynload, lib_dynload)
```

Comment:
Doesn't this need to be symlink(target_lib_dynload, lib_dynload, use_abspath_source=False) to make sure it doesn't get fiddled with...

Reply:
No, we want to avoid hardcoding the path to the installation directory in the symlink (we had a good reason to use `../..` rather than using the full path even)